### PR TITLE
Add SQL support content to Oracle connector

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/oracle.rst [hunk 1]
```diff
@@ -254,6 +254,22 @@ For Oracle ``NUMBER`` (without precision and scale), you can change
 
 .. _datetime mapping:
 
+.. _sqlserver-sql-support:
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/oracle.rst:257]: should be oracle, not sqlserver

**Review by jhlodin — APPROVED** [review]

File: docs/src/main/sphinx/connector/oracle.rst (hunk 1 above)
Comment (Jessie212) [reply docs/src/main/sphinx/connector/oracle.rst]: I corrected this while you were reviewing. Already amended.

**Review by hashhar — COMMENTED** [review]
See https://github.com/trinodb/trino/pull/8667#discussion_r677171808

File: docs/src/main/sphinx/connector/oracle.rst [hunk 2]
```diff
@@ -254,6 +254,22 @@ For Oracle ``NUMBER`` (without precision and scale), you can change
...
+Oracle. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :ref:`sql-schema-table-management`
+
+.. include:: sql-delete-limitation.fragment
+
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/oracle.rst:272]: Move to the bottom of the article, replacing the old "these statements are not supported" section

**Review by jhlodin — CHANGES_REQUESTED** [review]
Need to remove deprecated "unsupported statements" content

File: docs/src/main/sphinx/connector/oracle.rst [hunk 3]
```diff
@@ -254,6 +254,22 @@ For Oracle ``NUMBER`` (without precision and scale), you can change
...
+-----------
+
+The connector provides read access and write access to data and metadata in
+Oracle. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :ref:`sql-schema-table-management`
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/oracle.rst:269]: * :doc:`/sql/create-table`
* :doc:`/sql/create-table-as`
* :doc:`/sql/drop-table`
* :doc:`/sql/create-schema`
* :doc:`/sql/drop-schema`
* :doc:`/sql/alter-schema`
* :doc:`/sql/comment`

File: docs/src/main/sphinx/connector/oracle.rst [hunk 4]
```diff
@@ -254,6 +254,27 @@ For Oracle ``NUMBER`` (without precision and scale), you can change
...
+supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/create-schema`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/comment`
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/oracle.rst:274]: These 3 (CREATE SCHEMA, DROP SCHEMA and COMMENTS) are not supported in Oracle connector.

File: docs/src/main/sphinx/connector/oracle.rst (hunk 2 above)
Comment (Jessie212) [reply docs/src/main/sphinx/connector/oracle.rst]: Okay.

**Review by hashhar — APPROVED** [review]

**Review by jhlodin — APPROVED** [review]

**Review by rosewms — APPROVED** [review]
LGTM

File: docs/src/main/sphinx/connector/oracle.rst (hunk 4 above)
Comment (Jessie212) [reply docs/src/main/sphinx/connector/oracle.rst]: Updated here: https://github.com/trinodb/trino/pull/8844

**Pull request merged**